stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>This might seem obvious, but for readers aren't familiar with this stuff, I think it's worth explaning what "NaN" means in some of the columns.</p><p> </p><p>I also think it might be helpful to give an example by way of explanation of what this data shows. Something like: "From this data we can see the results of matches between named teams. For example in the first row we can see that Team Los Angeles Valiant won the round against Team San Francisco Shock on the Dorado map having scored..."</p><p> </p><p>This doesn't just help understand the data, it also helps make the dry data feel a bit more alive (there is, after all, very real emotion behind this data).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done</p>